providers/Mora/mod.test.ts

```diff
@@ -0,0 +1,59 @@
+import { describeProvider, makeProviderOptions } from '@/providers/test_spec.ts';
+import { stubProviderLookups } from '@/providers/test_stubs.ts';
+import { afterAll, describe } from '@std/testing/bdd';
+import { assertSnapshot } from '@std/testing/snapshot';
+
+import MoraProvider from './mod.ts';
+import { assertEquals } from 'std/assert/assert_equals.ts';
+
+describe('Mora provider', () => {
+	const bc = new MoraProvider(makeProviderOptions());
+	const lookupStub = stubProviderLookups(bc);
+
+	describeProvider(bc, {
+		urls: [{
+			description: 'package page',
+			url: new URL('https://mora.jp/package/43000006/00602488058599/'),
+			id: { type: 'package', id: '43000006/00602488058599' },
+			isCanonical: true,
+		}, {
+			description: 'album page with tracking parameter',
```

Comment:
Since I'm already nitpicking on this file, otherwise I wouldn't bother:
```suggestion
			description: 'package page with tracking parameter',
```